Mathlib/Algebra/Module/BigOperators.lean

```diff
@@ -54,6 +54,6 @@ lemma sum_piFinset_apply (f : κ → α) (s : Finset κ) (i : ι) :
   classical
   rw [Finset.sum_comp]
   simp only [eval_image_piFinset_const, card_filter_piFinset_const s, ite_smul, zero_smul, smul_sum,
-    sum_ite_mem, inter_self]
```

Comment:
What was this referring to before?

Reply:
(at a glance, this looks like a duplicate lemma which has a bad name?)

Reply:
[Finset.sum_ite_mem](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/BigOperators/Group/Finset.html#Finset.sum_ite_mem)

Reply:
The new one is `Fintype.`, the old one is `Finset.`, and so with both Fintype and Finset open `sum_ite_mem` becomes ambiguous

Reply:
Thanks, edited the description to make this clearer.